go-controller/Makefile

```diff
@@ -76,3 +79,14 @@ ifeq ($(CONTAINER_RUNNABLE), 0)
 else
 	@./hack/verify-gofmt.sh
 endif
+
+pkg/nbdb/ovn-nb.ovsschema:
+	curl -sSL https://raw.githubusercontent.com/ovn-org/ovn/$(OVN_VERSION)/ovn-nb.ovsschema -o $@
+
+pkg/sbdb/ovn-sb.ovsschema:
+	curl -sSL https://raw.githubusercontent.com/ovn-org/ovn/$(OVN_VERSION)/ovn-sb.ovsschema -o $@
+
+.PHONY: generate
+generate: pkg/nbdb/ovn-nb.ovsschema pkg/sbdb/ovn-sb.ovsschema
```

Comment:
there is already a codegen stage, I think we can merge all the code generator code there